config/local_ssl/localhost.key

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
same with this file -- moved to config/certs

Reply:
Done in 4ee6b23.